Tests/AppTests/Util+Snapshotting.swift

```diff
@@ -17,6 +17,14 @@ extension Snapshotting where Value == () -> HTML, Format == String {
     }
 }
 
+extension Snapshotting where Value == () -> Node<HTML.BodyContext>, Format == String {
+    public static var html: Snapshotting {
+        Snapshotting<String, String>.lines.pullback { node in
+            Current.siteURL = { "http://localhost:8080" }
+            return node().render(indentedBy: .spaces(2))
+        }
+    }
+}
```

Comment:
There must be a way to do this without duplicating this code, but it is beyond my abilities.

Reply:
We should be able to `pullback` to/from the existing strategy. I’ll take a look tomorrow!